js/src/Ice/ObjectAdapter.js

```diff
@@ -570,35 +533,6 @@ export class ObjectAdapter {
     }
 
     setState(state) {
```

Comment:
Maybe we can just remove this method and just do `this._state = state` in the few places we called it.

Reply:
Fixed as suggested.